### PR TITLE
Update all.json from Allure Security findings

### PR DESCRIPTION
**Pull request by michalisFr**
These are the new sites for today and the corresponding top-level domains added to `all.json`.
Please keep in mind that the script checks against all.json from the master branch. If there are open PRs, there may be duplicates.
Also, please check if the domains added are any well-known that shouldn't be blacklisted at the top level (eg. netlify, ddns, etc) and change these to include the subdomain.

Please check them for eligibility and liveness, add the additional information (urlscan, screenshot) were applicable, and separate them in eligible, non-eligible and dead.
The curator should only copy the "eligible" ones into the spreadsheet.

Below are 30 new scam sites, along with 29 domains added to `all.json`:

| Site | Domain |
|------|--------|
| debugstake.com | debugstake.com |
| wdsec6ure.com | wdsec6ure.com |
| wdsec6ure.drklt5.us | drklt5.us |
| openchainwz5.com | openchainwz5.com |
| webwalletdapp.co | webwalletdapp.co |
| openchainwz5.drklt5.us | N/A |
| validationdev.com | validationdev.com |
| wallet-connect.exalica.ba | exalica.ba |
| debugbyte.org | debugbyte.org |
| sysconnects.ml | sysconnects.ml |
| callingarray.com | callingarray.com |
| trushwallet.xyz | trushwallet.xyz |
| annexhive.com | annexhive.com |
| walletconnect.orisunewa.com | orisunewa.com |
| thecooglobal.com | thecooglobal.com |
| datavalidator.tk | datavalidator.tk |
| syncfixdapp.com | syncfixdapp.com |
| debugsurvey.com | debugsurvey.com |
| m.meta-extension-rectification.com | meta-extension-rectification.com |
| smartfix-dapps.com | smartfix-dapps.com |
| ransoit.com | ransoit.com |
| debugvalidate.org | debugvalidate.org |
| reconnectwall.com | reconnectwall.com |
| support-wallets.com | support-wallets.com |
| clouddigitaltray.com | clouddigitaltray.com |
| connectreflux.co | connectreflux.co |
| stollen.firebaseapp.com | firebaseapp.com |
| dappsio.co | dappsio.co |
| debugreflex.com | debugreflex.com |
| innhouse.team | innhouse.team |

### Eligible
| Site | Urlscan | Screenshot |
|-------|-------|-------|
| wdsec6ure.com | https://urlscan.io/result/4b5f4c28-ac1b-4753-b2d8-6c5c6cc3a83d/ | https://user-images.githubusercontent.com/14356826/192056891-35a80b4a-7e33-49c7-80d1-0784a2d48196.png |
| wdsec6ure.drklt5.us | https://urlscan.io/result/31378ada-25d3-4b87-98cb-00d801b38766/ | https://user-images.githubusercontent.com/14356826/192056880-49640c0a-400c-40a1-9553-2e95d4f31b59.png |
| openchainwz5.com | https://urlscan.io/result/e6cdbb13-4966-4b08-b0a3-21e7dfbde87d/ | https://user-images.githubusercontent.com/14356826/192056872-385c60e4-d35e-4594-b9e3-83e3802ca8b0.png |
| webwalletdapp.co | https://urlscan.io/result/f5d8d97e-247f-4afb-bbb5-bd08a69188be/dom/ | https://user-images.githubusercontent.com/14356826/192056863-45608e54-d970-49dc-9350-605c1f61a575.png |
| openchainwz5.drklt5.us | https://urlscan.io/result/484066b4-07c9-4937-ac28-5a481d56dbd2/ | https://user-images.githubusercontent.com/14356826/192056856-8a7c4d0e-1579-48d4-bb49-987006ffc0dc.png |
| stollen.firebaseapp.com | https://urlscan.io/result/f11a7b01-d2a5-4a7d-98ed-e8381903918f/dom/ | https://user-images.githubusercontent.com/14356826/192056838-f73b840a-271d-4f52-9d8c-9c3c5c32ed37.png |

### Non-eligible
| Site |
|-------|
| validationdev.com |
| wallet-connect.exalica.ba |
| sysconnects.ml |
| thecooglobal.com |

### Dead
| Site |
|-------|
| debugbyte.org |
| callingarray.com |
| trushwallet.xyz | 
| annexhive.com | 
| walletconnect.orisunewa.com | 
| thecooglobal.com | 
| datavalidator.tk | 
| syncfixdapp.com | 
| debugsurvey.com | 
| m.meta-extension-rectification.com | 
| smartfix-dapps.com | 
| ransoit.com | 
| debugvalidate.org | 
| reconnectwall.com | 
| support-wallets.com | 
| clouddigitaltray.com | 
| connectreflux.co | 
| dappsio.co | 
| debugreflex.com | 
| innhouse.team |

### CODE REVIEW

**Comment by michalisFr** [comment]
@ical10 Here are some sites for you

**Review by ical10 — APPROVED** [review]

**Comment by jacogr** [comment]
Please don't list the top-levels - they need to be checked individually. We seem to have included the very valid firebaseapp.com in this PR. 

TL;DR -

- list the sub-domains
- check each top-level to see if it is blockable, if so, add it

**Comment by ical10** [comment]
Oops, sorry Jaco, must have missed it. Will be careful next time.
